Add support for imperative options in configuration files (#2040)
f3d-app/f3d: doc/user/CONFIGURATION_FILE.md

Boolean options that have been turned on in the configuration file can be turned
off on the command line if needed, eg: `--point-sprites=false`.

### Imperative Options

Command line options and options that are changed interactively overrides options that are set in configuration files.

> This is not always a wanted behavior, so to force an option to always be taken into account even if set in command line are changed interactively, it is possible to use imperative options, by adding a `!` in front of the option name, eg:

Reviewer: ```suggestion
This is not always a desired behavior, so in order to force an option to always be taken into account even if set in command line or changed interactively, it is possible to use imperative options, by adding a `!` in front of the option name, eg:
```
